.env.production

```diff
@@ -0,0 +1,18 @@
+DFE_SIGN_IN_ISSUER=https://pp-oidc.signin.education.gov.uk:443
+DFE_SIGN_IN_REDIRECT_BASE_URL=https://localhost:3000
+DFE_SIGN_IN_IDENTIFIER=teacherpayments
+
+DFE_SIGN_IN_API_CLIENT_ID=teacherpayments
+DFE_SIGN_IN_API_ENDPOINT=https://pp-api.signin.education.gov.uk
+
+DQT_CLIENT_HOST=https://s118d01funcappfapp.azurewebsites.net/
+
+ENVIRONMENT_NAME=review
```

Comment:
@nomasprime Why is the environment called `review`?

Reply:
Seeds.